enhancements/management-workload-partitioning.md

```diff
@@ -215,7 +215,7 @@ does receive the content of annotations on the pod. Therefore, we will
 copy the original CPU requests for containers in management workload
 pods into a annotations that CRI-O can use to configure the CPU shares
 when running the containers. We will use the naming convention
-`io.openshift.workload.{workload-type}.cpushares/{container-name}` and
+`cpushares.{workload-type}.workload.openshift.io/{container-name}` and
```

Comment:
{workload-type}.cpushares.workload.openshift.io/{container-name}` probably makes more sense so the workload type won't think it claims all of foo.workload.openshfit.io

Reply:
We didn't want CRI-O to have to know about the annotation scheme, since the scheme is specific to OpenShift and CRI-O is not. So, we wanted the "resource type" (cpushares, whatever) to be tacked onto the front or end of the annotation. That originally led to the name being reversed.

I think I like #741 better, since it makes the resource type handling more transparent, but I want to hear what @haircommander thinks before we choose.